package.xml

```diff
@@ -5,27 +5,29 @@
 
   <description>A Ribbon for FreeCAD</description>
 
-  <version>0.0.2</version>
+  <version>1.0.0</version>
 
   <date>2024-07-28</date>
 
-  <maintainer>geolta</maintainer>
+  <maintainer>Paul Ebbers, Geolta</maintainer>
 
-  <license file="LICENSE">GPL v3.0</license>
+  <license file="LICENSE">GPL-3.0 license</license>
 
   <pythonmin>3.8.0</pythonmin>
 
-  <url type="website">https://github.com/geolta/FreeCAD-Ribbon/wiki</url>
+  <url type="website">https://github.com/APEbbers/FreeCAD-Ribbon/wiki</url>
 
-  <url branch="main" type="repository">https://github.com/geolta/FreeCAD-Ribbon</url>
+  <url branch="main" type="repository">https://github.com/APEbbers/FreeCAD-Ribbon</url>
```

Comment:
Here as well

Reply:
Can you change this when you merged all the changes to your repo? I prefer to have not to have links to another repo when I'm using my own repo for install